### PR TITLE
Splitting programs and adding a new tab

### PR DESCRIPTION
**Pull request by guillaumemoby**
Before sending your pull-request 💌 make sure :

- [ ] The tests are up-to-date and your code is tested

### CODE REVIEW

File: src/Components/HomePage.js [hunk 1]
```diff
@@ -134,6 +137,21 @@ const Theory = (props) => {
...
 };
 
+const myTeam = (props) => {
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.editorContainer}>
+        <TouchableHighlight onPress={() => props.navigation.navigate('PlayEditorPage')}>
+          <View style={styles.center}>
+            <Ionicons name="md-clipboard" color={theme.COLOR_PRIMARY} size={50} />
+            <Text style={styles.titleEditor}>Playbook</Text>
```
Comment (sallesma) [inline src/Components/HomePage.js:147]: Use i18n

File: src/Fixtures/config.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export const DrillTypes = {
   FITNESS: 'fitness',
   FRISBEE: 'frisbee',
+  FRISBEEJUNIOR: 'frisbeejunior',
```
Comment (sallesma) [inline src/Fixtures/config.js:4]: I think it should be a separate configuration, such as `AgeCatefory` with values `junior` or `senior` ?

(on fitness programs we use the EquipmentLabel to filter)

File: src/utils/locales/fr.js [hunk 3]
```diff
@@ -86,9 +86,11 @@ export default {
     frisbeeTab: 'Frisbee',
     fitnessTab: 'Fitness',
     theoryTab: 'Théorie',
+    teamTab: 'Mon Équipe',
     editor: 'Éditeur',
     drills: 'Exercices',
-    programs: "Programmes d'entraînement",
+    programs: 'Programmes pour Adultes',
```
Comment (sallesma) [inline src/utils/locales/fr.js:92]: Maybe we should use subtitle just like we did on fitness?

File: src/Components/HomePage.js [hunk 4]
```diff
@@ -134,6 +137,21 @@ const Theory = (props) => {
   );
 };
 
+const myTeam = (props) => {
+  return (
+    <View style={styles.mainContainer}>
+      <View style={styles.editorContainer}>
```
Comment (sallesma) [inline src/Components/HomePage.js:143]: It looks a bit "cheap" on the Team tab.
Is it a WIP layout that we should make better later, or a production-ready design?

Comment (guillaumemoby) [reply src/Components/HomePage.js:143]: It's a work in progress before we add the shared drills/videos...

File: src/Fixtures/config.js (hunk 2 above)
Comment (guillaumemoby) [reply src/Fixtures/config.js:4]: Good idea

File: src/utils/locales/fr.js (hunk 3 above)
Comment (guillaumemoby) [reply src/utils/locales/fr.js:92]: I have the feeling that it's clear enough like it

File: src/Fixtures/config.js (hunk 2 above)
Comment (guillaumemoby) [reply src/Fixtures/config.js]: I've tried to create a filter by age category but I couldn't make it working so far

**Comment by guillaumemoby** [comment]
Finally, I have other design in mind.

**Pull request closed without merging**